jenkinsfile-runner/Dockerfile

```diff
@@ -1,17 +1,14 @@
-FROM maven:3.5 as maven
-
-RUN git clone -b 1.0-beta-7 https://github.com/jenkinsci/jenkinsfile-runner && \
-    cd jenkinsfile-runner && \
-    mvn package --batch-mode -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
-
 FROM ppiper/jenkins-master
 
 USER root
+RUN curl -O https://repo.jenkins-ci.org/releases/io/jenkins/jenkinsfile-runner/jenkinsfile-runner/1.0-beta-7/jenkinsfile-runner-1.0-beta-7-app.zip && \
+    unzip jenkinsfile-runner-1.0-beta-7-app.zip && \
```

Comment:
Maybe we should use fixed destinations. Otherwise with relative paths a workdir command can easily break the build or external projects relying on this location.

Reply:
Agree, I did change that.